upload/src/addons/SV/StandardLib/InstallerHelper.php

```diff
@@ -418,6 +420,55 @@ protected function addOrChangeColumn(AbstractDdl $table, string $name, string $t
         throw new \LogicException('Unknown schema DDL type ' . \get_class($table));
     }
 
+    /**
+     * @since 1.10.0
+     *
+     * @param AbstractDdl $table
+     *
+     * @return void
+     */
+    protected function revertTableAlters(AbstractDdl $table)
```

Comment:
Should just change the type hint to `Alter` instead of throwing `\LogicException`

Reply:
This function doesn't look to be resetting `autoIncrementIndex` either.